Library/OAuth.swift

```diff
@@ -55,4 +65,67 @@ public struct OAuth {
       return nil
     }
   }
+
+  internal static func handleRedirect(
```

Comment:
Pulled this out into its own method to make it testable, without needing to deal with `ASWebAuthenticationSession` in the tests.